internal/cmd/egctl/testdata/translate/out/echo-gateway-api.cluster.yaml

```diff
@@ -0,0 +1,55 @@
+HTTPRoutes:
```

Comment:
can this be small case ?

Reply:
Do you mean `HttpRoutes`? It seems the official name is HTTPRoute: https://gateway-api.sigs.k8s.io/concepts/api-overview/#httproute

Reply:
I meant `httpRoutes`